<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>627</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveAs"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="separator"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="separator"/>
    <addaction name="action_SelecteAll"/>
    <addaction name="separator"/>
    <addaction name="action_Find"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="action_Close"/>
    <addaction name="action_Close_All"/>
    <addaction name="separator"/>
    <addaction name="action_Tile"/>
    <addaction name="action_Cascade"/>
    <addaction name="separator"/>
    <addaction name="action_Next"/>
    <addaction name="action_Previous"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Help"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="fileToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="editToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Undo"/>
   <addaction name="action_Redo"/>
   <addaction name="action_Cut"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Paste"/>
   <addaction name="action_Find"/>
  </widget>
  <widget class="QToolBar" name="windowToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Close"/>
   <addaction name="action_Previous"/>
   <addaction name="action_Next"/>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset resource="MaxscriptEditor.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Create a new file</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="MaxscriptEditor.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open an existing file</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="MaxscriptEditor.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save...</string>
   </property>
   <property name="toolTip">
    <string>Save the document to disk</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="text">
    <string>&amp;Save as...</string>
   </property>
   <property name="toolTip">
    <string>Save the document under a new name</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="MaxscriptEditor.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit the application</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="icon">
    <iconset resource="MaxscriptEditor.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="icon">
    <iconset resource="MaxscriptEditor.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset resource="MaxscriptEditor.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut the current selection's contents to the clipboard</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="MaxscriptEditor.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy the current selection's contents to the clipboard</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset resource="MaxscriptEditor.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste the clipboard's contents into the current selection</string>
   </property>
  </action>
  <action name="action_SelecteAll">
   <property name="text">
    <string>&amp;Select All</string>
   </property>
  </action>
  <action name="action_Find">
   <property name="icon">
    <iconset resource="MaxscriptEditor.qrc">
     <normaloff>:/images/find_normal.png</normaloff>:/images/find_normal.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Find/Replace</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="icon">
    <iconset resource="MaxscriptEditor.qrc">
     <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>Cl&amp;ose</string>
   </property>
   <property name="toolTip">
    <string>Close the active window</string>
   </property>
  </action>
  <action name="action_Close_All">
   <property name="text">
    <string>Close &amp;All</string>
   </property>
  </action>
  <action name="action_Tile">
   <property name="text">
    <string>&amp;Tile</string>
   </property>
  </action>
  <action name="action_Cascade">
   <property name="text">
    <string>&amp;Cascade</string>
   </property>
  </action>
  <action name="action_Next">
   <property name="icon">
    <iconset resource="MaxscriptEditor.qrc">
     <normaloff>:/images/go-next-view.png</normaloff>:/images/go-next-view.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Next window</string>
   </property>
  </action>
  <action name="action_Previous">
   <property name="icon">
    <iconset resource="MaxscriptEditor.qrc">
     <normaloff>:/images/go-previous-view.png</normaloff>:/images/go-previous-view.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Previous</string>
   </property>
   <property name="toolTip">
    <string>Previous window</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="icon">
    <iconset resource="MaxscriptEditor.qrc">
     <normaloff>:/images/help-browser.png</normaloff>:/images/help-browser.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Help</string>
   </property>
   <property name="toolTip">
    <string>Show the Maxscript Editor's Help document</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>Show the Maxscript Editor's About box</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MaxscriptEditor.qrc"/>
 </resources>
 <connections/>
</ui>
